build-tools/automation/yaml-templates/run-msbuild-device-tests.yaml

```diff
@@ -76,8 +76,8 @@ jobs:
           useDotNet: ${{ eq(parameters.target_framework, parameters.dotnet_targetframework) }}
           testRunTitle: MSBuildDeviceIntegration On Device - macOS-NoNode - ${{ parameters.job_suffix }}
           testAssembly: $(System.DefaultWorkingDirectory)/bin/Test$(XA.Build.Configuration)/MSBuildDeviceIntegration/${{ parameters.target_framework }}/MSBuildDeviceIntegration.dll
-          nunitConsoleExtraArgs: --where "cat != Node-1 && cat != Node-2 && cat != Node-3  && cat != Node-4 && cat != SystemApplication && cat != TimeZoneInfo && cat != SmokeTests ${{ parameters.nunit_categories }}"
-          dotNetTestExtraArgs: --filter "TestCategory != Node-1 & TestCategory != Node-2 & TestCategory != Node-3 & TestCategory != Node-4 & TestCategory != TimeZoneInfo ${{ parameters.nunit_categories }}"
+          nunitConsoleExtraArgs: --where "cat != Node-1 && cat != Node-2 && cat != Node-3 && cat != Node-4 && cat != SystemApplication && cat != TimeZoneInfo && cat != Localization && cat != SmokeTests ${{ parameters.nunit_categories }}"
+          dotNetTestExtraArgs: --filter "TestCategory != Node-1 & TestCategory != Node-2 & TestCategory != Node-3 & TestCategory != Node-4 & TestCategory != TimeZoneInfo & TestCategory != Localization ${{ parameters.nunit_categories }}"
```

Comment:
There are some other lanes that look at the `SmokeTests` category:
https://github.com/xamarin/xamarin-android/blob/acfc1efe047a5ec12bf199418d56b1796d3758dc/build-tools/automation/azure-pipelines.yaml#L243-L256

Do those need to check the `Localization` category? Or are they probably fine?

Reply:
I think its fine since the Localization tests are not "tagged" as SmokeTests in this case.